tds-prize-tool/src/main.rs

```diff
@@ -0,0 +1,69 @@
+mod prize;
+mod rewards_earned;
+mod utils;
+
+use clap::{crate_description, crate_name, crate_version, value_t_or_exit, App, Arg};
+use solana_core::blocktree::Blocktree;
+use solana_core::blocktree_processor::process_blocktree;
+use solana_sdk::genesis_block::GenesisBlock;
+use std::path::PathBuf;
+use std::process::exit;
+
+fn main() {
+    solana_logger::setup();
+
+    let matches = App::new(crate_name!())
+        .about(crate_description!())
+        .version(crate_version!())
+        .arg(
+            Arg::with_name("ledger")
+                .short("l")
+                .long("ledger")
+                .value_name("DIR")
+                .takes_value(true)
+                .required(true)
+                .help("Use directory for ledger location"),
+        )
+        .arg(
+            Arg::with_name("starting_balance")
+                .long("starting_balance")
+                .value_name("SOL")
+                .takes_value(true)
+                .required(true)
+                .help("Starting balance of validators at the beginning of TdS"),
+        )
+        .get_matches();
+
+    let ledger_path = PathBuf::from(value_t_or_exit!(matches, "ledger", String));
+    let starting_balance_sol = value_t_or_exit!(matches, "starting_balance", f64);
+
+    let genesis_block = GenesisBlock::load(&ledger_path).unwrap_or_else(|err| {
+        eprintln!(
+            "Failed to open ledger genesis_block at {:?}: {}",
+            ledger_path, err
+        );
+        exit(1);
+    });
+
+    let blocktree = match Blocktree::open(&ledger_path) {
+        Ok(blocktree) => blocktree,
+        Err(err) => {
+            eprintln!("Failed to open ledger at {:?}: {}", ledger_path, err);
+            exit(1);
+        }
+    };
+
+    println!("Verifying ledger...");
+    match process_blocktree(&genesis_block, &blocktree, None, true, None) {
```

Comment:
Def want `verify_ledger = false`, verifying PoH in this context probably borders on paranoia